components/form-fields/help-types.jsx

```diff
@@ -7,7 +7,7 @@ class HelpTypesField extends React.Component {
     super(props);
     this.state = { options: [] };
   }
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
Swapped 'componentWillMount' with 'componentDidMount' as recommended by React

Reply:
We can't move `Service.helpTypes.get()...` to `constrcutor` as the call is async and we are not allowed to update `state` in `constructor`.

Reply:
What I said above is not true. Because

> Service.get() immediately returns a value (namely, a Promise) and code execution continues, so the constructor doesn't wait for Service.helpTypes.get().then(...) to finish.

I've updated the PR. 🙈